.github/workflows/integration.yml

```diff
@@ -49,11 +49,6 @@ jobs:
         run: |
           tar -cf ${UPSTREAM_TAR} dist
           gzip -f --best ${UPSTREAM_TAR}
-
```

Comment:
I am not sure about this. @bruno-fs @infinitewarp do you remember what this step did ?